src/main/java/cholog/supp/api/study/service/InvitationService.java

```diff
@@ -0,0 +1,24 @@
+package cholog.supp.api.study.service;
+
+import cholog.supp.api.study.dto.request.InvitationRequest;
+import cholog.supp.common.jwt.JwtUtils;
+import cholog.supp.db.study.StudyGroup;
+import cholog.supp.db.study.StudyGroupRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class InvitationService {
+
+    private final JwtUtils jwtUtils;
+    private final String URL = "/api/v1/group/join?token=";
+    private final StudyGroupRepository studyGroupRepository;
+
+    public String createInvitation(InvitationRequest invitationRequest) {
```

Comment:
1. 
```suggestion
    public String createInvitationLink(InvitationRequest invitationRequest) {
```
2. 
```suggestion
    public String createInvitationToken(InvitationRequest invitationRequest) {
```
`createInvitation` 말고 1, 2 중에는 어떤가요~~?

Reply:
음 링크 붙이는거 괜찮다! 결과물을 링크를 반환하는거라서 1번으로 했을 때 더 명확할거같아

Reply:
`InvitationService`가 `URL`이라는 상태 값을 갖고 있으니까 `createInvitationLink`라고 명확하게 표현하는 것에 대해서 나도 찬성!